monitoring/healthz/Dockerfile

```diff
@@ -6,6 +6,10 @@ ENV WORKDIR /app
 WORKDIR ${WORKDIR}
 
 COPY package.json package-lock.json ./
+
+# Quick fix for CI where we don't run as root user
```

Comment:
why needed on these containers?

Reply:
These containers run npm in the dockerfile (so as root) and then execute as user